src/package.xml

```diff
@@ -2537,6 +2537,7 @@
         <members>DataImport__c.To_Be_Imported</members>
         <members>Engagement_Plan_Template__c.All</members>
         <members>General_Accounting_Unit__c.All</members>
+        <members>Grant_Deadline__c.All</members>
```

Comment:
@davisagli I believe this adds the "All" listview for the GrantDeadline object to the main NPSP Package, however the package doesn't include a Tab for this object. I suspect this may have been added accidentally.

Reply:
@force2b thanks, I missed taking this back out when I moved the tab out of the package because I hadn't quite made the connection that list views are only accessible in the context of a tab. I moved it.